src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionPoolKey.cs

```diff
@@ -33,12 +35,20 @@ internal override string ConnectionString
             }
         }
 
-        internal SqlConnectionPoolKey(string connectionString, SqlCredential credential, string accessToken, Func<SqlAuthenticationParameters, CancellationToken, Task<SqlAuthenticationToken>> accessTokenCallback) : base(connectionString)
+        internal SqlConnectionPoolKey(
+            string connectionString,
+            SqlCredential credential,
+            string accessToken,
+            Func<SqlAuthenticationParameters, CancellationToken,
+            Task<SqlAuthenticationToken>> accessTokenCallback,
+            Func<SSPIContextProvider> sspiContextProviderFactory
+            ) : base(connectionString)
```

Comment:
nit: I thiiiink the convention here is to put the closing paren on the same line as the last argument, then put `: base...` on the next line. Honestly kinda forget. But I'm sure there's an example somewhere else in the codebase